### PR TITLE
BaSP-44: Employee redux structure

### PR DESCRIPTION
**Pull request by DinoStampella**

### CODE REVIEW

File: src/Components/Employees/index.js [hunk 1]
```diff
@@ -1,23 +1,35 @@
 import styles from './employees.module.css';
-import { useState, useEffect } from 'react';
+// import { useState, useEffect } from 'react';
```
Comment (JulianFloresDev) [inline src/Components/Employees/index.js:2]: Delete Comments lines

File: src/Components/Employees/index.js [hunk 2]
```diff
@@ -1,23 +1,35 @@
...
+    dispatch(getEmployees(''));
   }, []);
+  // const [list, setList] = useState([]);
+  // const [isFetching, setIsFetching] = useState(true);
+  // useEffect(async () => {
+  //   // const response = await fetch(`${process.env.REACT_APP_API_URL}/employees`);
+  //   // const data = await response.json();
+  //   // setList(data.data || []);
+  //   // setTimeout(() => setIsFetching(false), 2000);
+  // }, []);
```
Comment (JulianFloresDev) [inline src/Components/Employees/index.js:22]: Delete this too

File: src/Components/Employees/index.js [hunk 3]
```diff
@@ -1,23 +1,35 @@
...
   return (
     <section className={styles.container}>
       {isFetching ? (
         <div className={styles.container}>
           <Spinner entitie="Employees" />
         </div>
+      ) : error ? (
+        <div>
+          <h2>404: server not found</h2>
+        </div>
```
Comment (JulianFloresDev) [inline src/Components/Employees/index.js:32]: Awasome!!!

File: src/Components/Share/Form/index.js [hunk 4]
```diff
@@ -38,74 +68,37 @@ const Form = () => {
...
-        history.push(`/${entitie}`);
-      }, 2000);
-    } catch (error) {
-      console.error(error);
+    dispatch(editItem({ ...itemToPUT }));
+    switch (entitie) {
+      case 'employees':
+        dispatch(editEmployee(id, body));
+        break;
+      default:
```
Comment (JulianFloresDev) [inline src/Components/Share/Form/index.js:81]: Nothing is happening by default!

**Review by JulianFloresDev — APPROVED** [review]
Well Done! It's ready for QA in my opinion

File: src/Components/Employees/index.js (hunk 3 above)
Comment (DinoStampella) [reply src/Components/Employees/index.js:32]: Your idea !!!

**Review by kolodani — APPROVED** [review]
great work!

**Review by JulianFloresDev — DISMISSED** [review]
Approved

**Review by MatiasPazConti — APPROVED** [review]

**Review by MatiasPazConti — APPROVED** [review]

**Review by JulianFloresDev — APPROVED** [review]
Approved again!!

**Review by LuisQueff — APPROVED** [review]
All correct, ready to merge

**Pull request merged**